Tests/DistributedActorsTests/TraversalTests.swift

```diff
@@ -82,7 +82,9 @@ final class TraversalTests: ActorSystemXCTestCase {
                 "system",
                 "receptionist",
                 "receptionist-ref",
-                // "gossip", // TODO: should it also be here?
+                // cluster actors ---
+                "downingStrategy", "swim", "nodeDeathWatcher", "gossip", "cluster", "leadership",
+                // end of cluster actors ---
```

Comment:
this test needs to account for the cluster actors now 👍 

Seems all others do pass, at least locally for me

Reply:
🙏